content/post/054-go-ipfs-0.4.22.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2019-08-06
+url: 054-go-ipfs-0.4.22
+title: go-ipfs 0.4.22 released
+author: Steven Allen
+---
+
+We're releasing a PATCH release of go-ipfs based on 0.4.21 containing some critical fixes.
+
+The IPFS network has scaled to the point where small changes can have a wide-reaching impact on the entire network. To keep this situation from escalating, we've put a hold on releasing new features until we can improve our [release process](https://github.com/ipfs/go-ipfs/pull/6482) (which we will be trialing in this release) and [testing procedures](https://github.com/ipfs/go-ipfs/issues/6483).
```

Comment:
The blog post will be published after the release so makes sense to refer to the trial in the past tense?

```suggestion
The IPFS network has scaled to the point where small changes can have a wide-reaching impact on the entire network. To keep this situation from escalating, we've put a hold on releasing new features until we can improve our [release process](https://github.com/ipfs/go-ipfs/pull/6482) (which we've trialed in this release) and [testing procedures](https://github.com/ipfs/go-ipfs/issues/6483).
```